Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -566,6 +566,37 @@ define([
         tweens.update();
     }
 
+    var pickGlobeScratchRay = new Ray();
+    var scratchDepthIntersection = new Cartesian3();
+    var scratchRayIntersection = new Cartesian3();
+
+    function pickGlobe(controller, mousePosition, result) {
+        var scene = controller._scene;
+        var globe = controller._globe;
+        var camera = scene.camera;
+
+        if (!defined(globe)) {
+            return undefined;
+        }
+
+        var depthIntersection;
+        if (defined(scene._globeDepth)) {
```

Comment:
Can we have a public helper so users can also see if this is supported?  The Sandcastle example should also check it.